src/OnboardingSPA/components/TextInput/TextInputSiteGen/index.js

```diff
@@ -90,6 +94,12 @@ const TextInputSiteGen = ( {
 						onChange={ ( e ) => onTextChange( e ) }
 					/>
 				</div>
+				{ getRemainingCharacterCount() > 0 && (
+					<p className={ 'nfd-sg-input-box__count' }>
+						{ getRemainingCharacterCount() }
+						{ __( ' Characters left', 'wp-module-onboarding' ) }
```

Comment:
Non-blocker: Can we use backticks and get this from a 'getContents' function that is our standard for text displayed in any component? Something like: `${remainingCharacterCount} Characters left`.

Reply:
Fixed.